src/iterative_sorting/iterative_sorting.py

```diff
@@ -6,20 +6,36 @@ def selection_sort( arr ):
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-             
-
-
-
+        for j in range(cur_index, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
+        
         # TO-DO: swap
-
-
-
+        temp = arr[smallest_index]
+        arr[smallest_index] = arr[i]
+        arr[i] = temp
 
     return arr
 
 
 # TO-DO:  implement the Bubble Sort function below
 def bubble_sort( arr ):
+    swaps = 0
+
+    while True:
+        for i in range(0, len(arr) - 1):
+            swaps = 0
+            if arr[i] > arr[i+1]:
+                temp = arr[i+1]
+                arr[i+1] = arr[i]
+                arr[i] = temp
+                swaps += 1
+                break
+        if swaps == 0:
```

Comment:
if swaps was a boolean, you could probably shave of a few lines of code in the bubble_sort